### PR TITLE
feat: add patchReleaseChannel endpoint

### PR DESCRIPTION
**Pull request by ybw0014**
I'm not sure if I missed something, but local env seems to work well.

patch request via Postman:
image

Get the project info after updating:
image

### CODE REVIEW

**Comment by Seggan** [comment]
Yay

File: worker/src/handlers/projects/project.ts [hunk 1]
```diff
@@ -185,3 +185,34 @@ export async function patchProjectSettings(ctx: Ctx, body: ProjectSettingsBody)
...
+
+	// Get release channel
+	const releaseChannels = await ReleaseChannelStore.getReleaseChannelsForProject(project.projectId);
+	const releaseChannel = releaseChannels.find(channel => channel.name === ctx.req.param('releaseChannel'));
+	if (releaseChannel === undefined) {
+		return errors.ReleaseChannelNotFound.toResponse(ctx);
+	}
+
+	await ReleaseChannelStore.updateReleaseChannel(releaseChannel.releaseChannelId, body);
+	return success('Release channel updated!', releaseChannel);
```
Comment (WalshyDev) [inline worker/src/handlers/projects/project.ts:217]:
```suggestion
	const updatedReleaseChannel = await ReleaseChannelStore.updateReleaseChannel(releaseChannel.releaseChannelId, body);

	return success('Release channel updated!', updatedReleaseChannel);
```

File: .gitignore [hunk 2]
```diff
@@ -5,3 +5,6 @@ dist/
 .turbo/
```
Comment (WalshyDev) [inline .gitignore:5]:
```suggestion
.turbo/
.vscode/
.idea/
```

File: .gitignore [hunk 3]
```diff
@@ -5,3 +5,6 @@ dist/
...
 
 .dev.vars
+
+.vscode/
+.idea/
```
Comment (WalshyDev) [inline .gitignore:10]:
```suggestion
```

**Review by WalshyDev — APPROVED** [review]

**Pull request merged**